lineman/app/components/inbox_page/inbox_page_controller.coffee

```diff
@@ -4,21 +4,26 @@ angular.module('loomioApp').controller 'InboxPageController', ($scope, $rootScop
   $rootScope.$broadcast('analyticsClearGroup')
 
   @threadLimit = 5
+  @views =
+    groups: {}
+
+  @loading = -> !(CurrentUser.inboxLoaded and CurrentUser.membershipsLoaded)
```

Comment:
Wait to initialize inbox until we have all of the current user's memberships (ie groups) and the inbox query is complete.